web/beacon-app/src/utils/misc.tsx

```diff
@@ -1 +1,12 @@
 export const StatusIconMap = {};
+
+export const isCurrentMenuPath = (href: string, pathname: string) => {
+  const hrefArray = href.split('/');
+  const pathnameArray = pathname.split('/');
+
+  if (hrefArray.length > 2 && pathnameArray.length > 2) {
```

Comment:
Should this effect also be applied to the Topics page?

Reply:
For now, it won't work. We'll create a story to handle that later since the topic starts with app/topics and the project starts with app/projects.